curvefs/src/mds/common/mds_define.h

```diff
@@ -39,6 +44,26 @@ typedef uint32_t UserIdType;
 
 const uint32_t UNINTIALIZE_ID = 0u;
 
+inline std::string BuildPeerId(const std::string &ip, uint32_t port,
```

Comment:
add some annotations to explain why we need build peer id, and its format

Reply:
> add some annotations to explain why we need build peer id, and its format

fixed